bundle/manifests/logging.openshift.io_clusterloggings_crd.yaml

```diff
@@ -3,8 +3,14 @@ kind: CustomResourceDefinition
 metadata:
   name: clusterloggings.logging.openshift.io
 spec:
+  additionalPrinterColumns:
+  - JSONPath: .spec.managementState
```

Comment:
What is this?

Reply:
See screenshot above. It exposes the spec field as part of `oc get clusterlogging instance`. It enables to check the CRs state without doing an `oc describe clusterlogging instance`.